src/select-menu/src/SelectMenuContent.js

```diff
@@ -68,26 +74,7 @@ export default class SelectMenuContent extends PureComponent {
           flexDirection="column"
           borderRight={hasDetailView ? 'muted' : null}
         >
-          {hasTitle && (
-            <Pane
-              display="flex"
-              alignItems="center"
-              borderBottom="default"
-              padding={8}
-              height={headerHeight}
-              boxSizing="border-box"
-            >
-              <Pane flex="1">
-                <Heading size={400}>{title}</Heading>
-              </Pane>
-              <IconButton
-                icon="cross"
-                appearance="minimal"
-                height={24}
-                onClick={close}
-              />
-            </Pane>
-          )}
+          {hasTitle && titleView}
```

Comment:
Hm, I think we should provide the below (removed) jsx as the _default prop_ value for `titleView`. That way consumers of this component can continue using it as is.

The way you've moved the code around makes this a breaking change, technically.